res/values-b+es+r419/strings.xml

```diff
@@ -835,7 +835,7 @@
     <string name="UNBLOCK">DESBLOQUEAR</string>
     <string name="Emotes">EMOTICONOS</string>
     <string name="Support">Soporte</string>
-    <string name="Already_Blocked">Ya bloqueado</string>
+    <string name="Already_Blocked">Bloqueado</string>
```

Comment:
This one isn't supposed to be changed like that, otherwise it ain't sound like "you already blocked". Please return it as it was.